content/tutorial-paychan.md

```diff
@@ -0,0 +1,611 @@
+            "validated": true
+        }
+    }
+
+In the response from the JSON-RPC, the payer should look for the following:
+
+- In the transaction's `meta` field, confirm that the `TransactionResult` is `tesSUCCESS`.
+- Confirm that the response has `"validated":true` to indicate the data comes from a validated ledger. (The result `tesSUCCESS` is only [final](reference-transaction-format.html#finality-of-results) if it appears in a validated ledger version.)
+- In the `AffectedNodes` array of the transaction's `meta` field, look for a `CreatedNode` object with the `LedgerEntryType` of `PayChannel`. The `LedgerIndex` field indicates the Channel ID. (In the above example, this is a hex string starting with "5DB0...") The Channel ID is necessary later to sign claims.
+
+
+## 2. The payee checks specifics of the payment channel.
+
+You can look up payment channels with the `account_channels` API method, using the payer of the channel, as in the following example (using the JSON-RPC API):
+
+Request:
+
+    POST http://localhost:5005/
+    Content-Type: application/json
+
+    {
+        "method": "account_channels",
+        "params": [{
+            "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+            "destination_account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            "ledger_index": "validated"
+        }]
+    }
+
+Response:
+
+    200 OK
+
+    {
+        "result": {
+            "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+            "channels": [{
+                "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+                "amount": "100000000",
+                "balance": "0",
+                "channel_id": "5DB01B7FFED6B67E6B0414DED11E051D2EE2B7619CE0EAA6286D67A3A4D5BDB3",
+                "destination_account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+                "destination_tag": 20170428,
+                "public_key": "aB44YfzW24VDEJQ2UuLPV2PvqcPCSoLnL7y5M1EzhdW4LnK5xMS3",
+                "public_key_hex": "023693F15967AE357D0327974AD46FE3C127113B1110D6044FD41E723689F81CC6",
+                "settle_delay": 86400
+            }],
+            "status": "success"
+        }
+    }
+
+The payee should check that the parameters of the payment channel are suitable for their specific use case, including all of the following:
+
+- Confirm the `destination_account` field has the payee's correct address.
+- Confirm the `settle_delay` field has a settlement delay in seconds that provides enough time for the payee to redeem outstanding claims.
+- Confirm the fields `cancel_after` (immutable expiration) and `expiration` (mutable expiration), if they are present, are not too soon. The payee should take note of these times so they can be sure to redeem claims before then.
+- Take note of the `public_key` and `channel_id` fields. These are necessary later to verify and redeem claims.
+- _(Optional)_ Confirm the `destination_tag` field, if present, has the correct destination tag (if desired).
```

Comment:
nitpick: Using "Optional" and "(if desired)" seems a little redundant. One or the other seems sufficient.